### DISCUSSION TITLE
Build/Buldoze Selection Colour

### DISCUSSION THREAD

**Discussion by Phelzier** [voxeltycoon/issues#390] [UI&UX]
It might be me, but I think that the colour to see what button is selected isn't very clear.  Could you do something a little better to show when a mode is selected.

The bulldozer is selected in the picture below.

![image](https://user-images.githubusercontent.com/22050004/60357041-7bbdf180-99ca-11e9-9ee9-58a0b9a14951.png)

**Comment by victorleblais** [comment]
Maybe making the background red when selected could help.

Also, adding something like a red outline around the screen would be very helpful too. Other games like Cities Skyline / Sim City / Sims do something similar to make extra-sure you know you're in Buildoze mode.

Edit: Here's an example of what I was thinking of:
![image](https://user-images.githubusercontent.com/4365075/60377838-607ad280-9a1a-11e9-9b68-4b1e4131aa29.png)

**Comment by victorleblais** [comment]
Fixed by 0.88.5 !
https://voxeltycoon.xyz/devlog/update-88-5

Like we both suggested, the icon is now red, and there's a red frame all around the screen.

![image](https://github.com/voxeltycoon/issues/assets/4365075/6c60cb49-ebb2-4a73-a15d-e4290b6897a1)
